docs/build.ps1

```diff
@@ -1,22 +1,33 @@
+Write-Host "Branch: $env:APPVEYOR_REPO_BRANCH"
+Write-Host "PR Number: $env:APPVEYOR_PULL_REQUEST_NUMBER"
+Write-Host "Job Name: $env:APPVEYOR_JOB_NAME"
+if (-not (Test-Path env:git_access_token)) {
```

Comment:
why not just `if( $env:git_access_token )`

Reply:
Fixed.